txt2hpo/extract.py

```diff
@@ -320,12 +352,12 @@ def self_evaluation(correct_spelling=False, resolve_conflicts=False):
     logger.info('Running self evaluation, this may take a few minutes \n')
     i = 0
     n_nodes = len(hpo_network.nodes)
-
+    hpo = Extractor(correct_spelling=correct_spelling,resolve_conflicts=resolve_conflicts).hpo
     for node in hpo_network:
         total += 1
         term = hpo_network.nodes[node]['name']
         hpids = []
-        extracted = hpo(term, correct_spelling=correct_spelling, resolve_conflicts=resolve_conflicts)
+        extracted = hpo(term)
```

Comment:
`extractor.hpo(term)`

Reply:
yeah, that seems good, I just wanted to fit those into existing tests as much as possible